content/en/continuous_integration/tests/_index.md

```diff
@@ -35,23 +35,23 @@ cascade:
+| {{< ci-details title="Accurate time/durations results" >}}Microseconds resolution in time and duration results. This is usually not the case for JUnit XML reports.{{< /ci-details >}}         |  ✓   |  ✓   |            ✓             |   ✓    |  ✓   |   ✓   |            ✗             |
+| {{< ci-details title="Structured stack traces" >}} {{< /ci-details >}}                 |  ✓   |  ✓   |            ✓             |   ✓    |  ✓   |   ✓   |            ✗             |
+| {{< ci-details title="Distributed traces on integration tests" >}}Tests that make calls to external services instrumented with Datadog will show the full distributed trace in its test details.{{< /ci-details >}} |  ✓   |  ✓   |            ✓             |   ✗    |  ✓   |   ✓   |            ✗             |
+| {{< ci-details title="Agent-based reporting" >}}Ability to report test information via the Datadog Agent.{{< /ci-details >}}                   |  ✓   |  ✓   |            ✓             |   ✓    |  ✓   |   ✓   |            ✗             |
+| {{< ci-details title="Agentless reporting" >}}Ability to report test information without the need of the Datadog Agent.{{< /ci-details >}}                     |  ✓   |  ✓   |            ✓             |   ✓    |  ✗   |   ✓   |            ✓             |
+| {{< ci-details title="Test suite level visibility" >}}Visibility over the whole testing process, including session, module, suites and tests.{{< /ci-details >}}             |  ✓   |  ✓   |            ✓             |   ✓    |  ✗   |   ✓   |            ✓             |
+| {{< ci-details title="Manual API" >}}Ability to create CI Visibility events for test frameworks that are not currently supported by Datadog's automatic instrumentation.{{< /ci-details >}}                              |  ✓   |  ✓   |            ✓             |   ✗    |  ✗   |   ✓   |            ✗             |
+| {{< ci-details title="Codeowner by test" >}}Automatic detection of the owner of a test based on the CODEOWNERS file.{{< /ci-details >}}                       |  ✓   |  ✓   |            ✓             |   ✗    |  ✗   |   ✓   |            ✗             |
+| {{< ci-details title="Source code start/end" >}}Automatic report of the start and end line of a test.{{< /ci-details >}}         |  ✓   |  ✓   | ✓&nbsp;(only&nbsp;start) |   ✗    |  ✗   |   ✓   | ✓&nbsp;(only&nbsp;start) |
+| {{< ci-details title="CI and git info" >}}Automatic collection of git and CI environment metadata, such as CI provider, git commit SHA or pipeline URL.{{< /ci-details >}}                         |  ✓   |  ✓   |            ✓             |   ✓    |  ✓   |   ✓   |            ✓             |
+| {{< ci-details title="Git metadata upload" >}}Automatic upload of git tree information used for Intelligent Test Runner.{{< /ci-details >}}                     |  ✓   |  ✓   |            ✓             |   ✓    |  ✗   |   ✓   |            ✓             |
```

Comment:
I think we can add a link to Intelligent Test Runner docs page here (and in the next row too).